backend/comments/models.py

```diff
@@ -20,10 +22,11 @@ class Visibility(TextChoices):
 
 class Comment(Model):
     type =  CharField(blank=False, null=False, default="comment", max_length=200)
-    id = URLField(primary_key=True, blank=True, null=False)
+    id = URLField(blank=True, null=False)
+    uuid = UUIDField(primary_key=True, default=uuid4, editable=False)
     # when all row in author is deleted, this comment will be too
     author = models.ForeignKey(Author,blank=False, null=True, on_delete=CASCADE)
-    contentType = CharField(default=ContentType.markdown, blank=False, null=False, choices=ContentType.choices, max_length=200)
+    contentType = CharField(default=ContentType.plain, blank=False, null=False, choices=ContentType.choices, max_length=200)
```

Comment:
Should just be text/markdown, no choices. Do `python3 manage.py makemigrations` and `python3 manage.py migrate` once you make this change.

```suggestion
    contentType = CharField(default='text/markdown', blank=False, null=False, max_length=200)
```

Reply:
Hi, i have changed it